apps/jenkins/jenkins-webhook-relay/jenkins-webhook-relay.yaml

```
@@ -0,0 +1,22 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: jenkins-webhook-relay
+spec:
+  chart:
+    spec:
+      chart: ./charts/jenkins-webhook-relay
+      interval: 1m
+      reconcileStrategy: ChartVersion
+      sourceRef:
+        kind: GitRepository
+        name: jenkins-webhook-relay
+        namespace: flux-system
+      version: '*'
+      valuesFiles:
+        - ./charts/jenkins-webhook-relay/values.yaml
```

values are always picked default
```suggestion
      valuesFiles:
        - ./charts/jenkins-webhook-relay/values.yaml
```